packages/service-clients/end-to-end-tests/azure-client/.eslintrc.cjs

```diff
@@ -3,12 +3,41 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * Copied over from common/build/eslint-config-fluid/minimal-deprecated.js,
+ * since we need to add more entries in this package but don't want to lose these.
+ */
+const permittedImports = [
```

Comment:
Can we export this from `eslint-config-fluid` and just extend it here?

Reply:
I thought we had done that recently but couldn't find it, maybe hasn't merged. I was leaning towards not waiting for that, which has to be merged, released, and consumed before we can leverage it here.

Reply:
Added a TODO for when it happens though

Reply:
Resolving as not applicable anymore.